apis/goharbor.io/v1alpha3/harborcluster_webhook.go

```diff
@@ -66,7 +67,7 @@ func (hc *HarborCluster) ValidateUpdate(old runtime.Object) error {
 
 	obj, ok := old.(*HarborCluster)
 	if !ok {
-		return fmt.Errorf("failed type assertion on kind: %s", old.GetObjectKind().GroupVersionKind().String())
+		return errors.Errorf("failed type assertion on kind: %s", old.GetObjectKind().GroupVersionKind().String())
```

Comment:
That way to handle the `goerr113` linter looks like a cheat.
It seems that this linter detect that `fmt.Errorf()` is untyped error that is not a *normal* case.
And it does not detect the `errors.Errorf()` which does the same thing.
Am I wrong? WDYT?

Reply:
yes, I just reference the note in https://github.com/Djarvur/go-err113#reports, it tells this package would be ignored.
anyway in future it can be refactor again to define some package level errors and use `%w` in Errorf()